components/modal.css

```diff
@@ -0,0 +1,45 @@
+.modal h1 {
+  font-size: 3rem;
+  font-weight: bold;
+}
+.demo-modal {
+  width: fit-content;
+  height: 10rem;
+  background-color: white;
```

Comment:
you should use variable for the color for ease to use and replace in case needed at only one place

Reply:
Done.